docs/how-to/datapath-only-mode.rst

```diff
@@ -0,0 +1,112 @@
+==================
+Datapath-only Mode
+==================
+
+This is a mode of operation in which MicroOVN cluster runs only ``switch`` and
+``chassis`` service, with ``central`` service being provided externally by another
+MicroOVN cluster or by some other OVN deployment.
```

Comment:
It is pertinent to put emphasis on the main use case being migration / transition.

In the current wording a unsuspecting user might mistake this feature to OVN interconnect or similar where two deployments actually can work together, which is not the case for this particular feature.